### PR TITLE
Restructure Archive Doc Page - Document Latest Options

### PR DESCRIPTION
**Pull request by pcarioufr**
### Motivation

Mostly restructuring and rewording. And better connection with RBAC-related content.

### Preview

https://docs-staging.datadoghq.com/pcarioufr/log_archives_setup_refacto/logs/archives

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/archives/_index.md [hunk 1]
```diff
@@ -17,94 +17,258 @@ further_reading:
...
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+### Route your Logs to a Bucket
+
+Go to your [Archives page][99] in Datadog and select the **Add a new archive** option at the bottom. 
+
+Only Datadog users with [Logs Write Archive permission][98] can complete this and the following step.
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:181]:
```suggestion
Only Datadog users with [LOGS_WRITE_ARCHIVES permission][98] can complete this and the following step.
```

File: content/en/logs/archives/_index.md [hunk 2]
```diff
@@ -17,94 +17,258 @@ further_reading:
...
+
+[98]: /account_management/rbac/permissions/?tab=ui#logs_write_archives
+[99]: https://app.datadoghq.com/logs/pipelines/archives
+
+
+{{< tabs >}}
+{{% tab "AWS S3" %}}
+
+Select the appropriate AWS account + role combination for your S3 bucket.
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:190]:
```suggestion
Select the appropriate AWS account and role combination for your S3 bucket. 
```

File: content/en/logs/archives/_index.md [hunk 3]
```diff
@@ -17,94 +17,258 @@ further_reading:
...
+[98]: /account_management/rbac/permissions/?tab=ui#logs_write_archives
+[99]: https://app.datadoghq.com/logs/pipelines/archives
+
+
+{{< tabs >}}
+{{% tab "AWS S3" %}}
+
+Select the appropriate AWS account + role combination for your S3 bucket. 
+
+Input your bucket name. **Optional**: input a prefix directory for all the content of your log archives.
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:192]:
```suggestion
Input your bucket name. **Optional**: Input a prefix directory for all the content of your log archives.
```

File: content/en/logs/archives/_index.md [hunk 4]
```diff
@@ -17,94 +17,258 @@ further_reading:
...
+
+{{< img src="logs/archives/logs_archive_aws_setup.png" alt="Set your S3 bucket info in Datadog"  style="width:75%;">}}
+
+{{% /tab %}}
+{{% tab "Azure Storage" %}}
+
+
+Select the **Azure Storage** archive type, and the Azure Tenant and Client for the Datadog App that has the Storage Blob Data Contributor role on your storage account.
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:200]:
```suggestion
Select the **Azure Storage** archive type, and the Azure tenant and client for the Datadog app that has the Storage Blob Data Contributor role on your storage account. 
```

File: content/en/logs/archives/_index.md [hunk 5]
```diff
@@ -17,94 +17,258 @@ further_reading:
...
+
+{{< img src="logs/archives/logs_archive_aws_setup.png" alt="Set your S3 bucket info in Datadog"  style="width:75%;">}}
+
+{{% /tab %}}
+{{% tab "Azure Storage" %}}
+
+
+Select the **Azure Storage** archive type, and the Azure Tenant and Client for the Datadog App that has the Storage Blob Data Contributor role on your storage account. 
+
+Input your storage account name and a container name for your archive. **Optional**: input a prefix directory for all the content of your log archives.
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:202]:
```suggestion
Input your storage account name and a container name for your archive. **Optional**: Input a prefix directory for all the content of your log archives.
```

File: content/en/logs/archives/_index.md [hunk 6]
```diff
@@ -17,94 +17,258 @@ further_reading:
...
+
+{{< img src="logs/archives/logs_archive_azure_setup.png" alt="Set your Azure storage account info in Datadog"  style="width:75%;">}}
+
+
+{{% /tab %}}
+{{% tab "Google Cloud Storage" %}}
+
+
+Select the **GCS** archive type, and the GCS Service Account that has permissions to write on your storage bucket.
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:211]:
```suggestion
Select the **GCS** archive type, and the GCS service account that has permissions to write on your storage bucket. 
```

File: content/en/logs/archives/_index.md [hunk 7]
```diff
@@ -17,94 +17,258 @@ further_reading:
...
+{{< img src="logs/archives/logs_archive_azure_setup.png" alt="Set your Azure storage account info in Datadog"  style="width:75%;">}}
+
+
+{{% /tab %}}
+{{% tab "Google Cloud Storage" %}}
+
+
+Select the **GCS** archive type, and the GCS Service Account that has permissions to write on your storage bucket. 
+
+Input your bucket name. **Optional**: input a prefix directory for all the content of your log archives.
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:213]:
```suggestion
Input your bucket name. **Optional**: Input a prefix directory for all the content of your log archives.
```

File: content/en/logs/archives/_index.md [hunk 8]
```diff
@@ -17,94 +17,258 @@ further_reading:
...
+{{% /tab %}}
+{{< /tabs >}}
 
 ### Advanced Settings
 
+#### Datadog Permissions
+
+By default:
+
+* all Datadog Admin users can create, edit and reorder (see [Configure Multiple Archives](#configure-multiple-archives) Archives,
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:226]:
```suggestion
* All Datadog Admin users can create, edit and reorder (see [Configure Multiple Archives](#configure-multiple-archives) archives
```

File: content/en/logs/archives/_index.md [hunk 9]
```diff
@@ -17,94 +17,258 @@ further_reading:
...
+
+## Configure an Archive
 
-## Create and configure a storage bucket
+### Set Integration
 
 {{< tabs >}}
 {{% tab "AWS S3" %}}
 
+If not already configured, set up the [AWS integration][1] for the AWS account that holds your S3 bucket.
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:45]: Getting a 404 for this integration link

File: content/en/logs/archives/_index.md [hunk 10]
```diff
@@ -17,97 +17,260 @@ further_reading:
...
+Configure your Datadog account to forward all the logs ingested - whether [indexed][1] or not - to a cloud storage system of your own. Keep your logs in a storage-optimized archive for longer periods of time and meet compliance requirements whilst also keeping auditability for ad hoc investigations, with [Rehydration][2].
 
-This guide shows you how to set up an archive for forwarding ingested logs to your own cloud-hosted storage bucket.
+{{< img src="logs/archives/log_archives_s3_multiple.png" alt="Archive page view"  style="width:75%;">}}
 
-**Note:** only Datadog users with admin status can create, modify, or delete log archive configurations.
+This guide shows you how to set up an archive for forwarding ingested logs to your own cloud-hosted storage bucket:
 
-{{< img src="logs/archives/log_archives_s3_multiple.png" alt="Archive page view"  style="width:75%;">}}
+1. Set up Datadog [Integration](#set-integration) for your cloud provider
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:28]:
```suggestion
1. Set up a Datadog [integration](#set-up-an-integration) for your cloud provider
```

File: content/en/logs/archives/_index.md [hunk 11]
```diff
@@ -17,97 +17,260 @@ further_reading:
...
+2. Create a [storage bucket](#create-a-storage-bucket)
+3. Set [permissions](#set-permissions) read and/or write on that archive
+4. [Route your logs](#route-your-logs-to-a-bucket) to and from that archive
+5. Configure [advanced settings](#advanced-settings) such as encryption, storage class, and tags
+6. [Validate](#validation) your setup checking for possible misconfigurations that Datadog would be able to detect for you
+
+**Note:** only Datadog users with [Logs Write Archive permssion][3] can create, modify, or delete log archive configurations.
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:35]:
```suggestion
**Note:** only Datadog users with [Logs Write Archive permission][3] can create, modify, or delete log archive configurations.
```

File: content/en/logs/archives/_index.md [hunk 12]
```diff
@@ -17,97 +17,260 @@ further_reading:
   text: "Logging without Limits*"
 ---
 
-## Archives
+## Overview
 
-Configure your Datadog account to forward all the logs ingested to a cloud storage system. Keeping your logs in a storage-optimized archive for longer periods of time enables you to meet compliance requirements, and to keep auditability for ad hoc investigations within budget. Once your logs are archived for the long term, [you have access to them][1] should you ever need to investigate something unexpected—or that might have happened a long time ago.
+Configure your Datadog account to forward all the logs ingested - whether [indexed][1] or not - to a cloud storage system of your own. Keep your logs in a storage-optimized archive for longer periods of time and meet compliance requirements whilst also keeping auditability for ad hoc investigations, with [Rehydration][2].
```
Comment (sarina-dd) [inline content/en/logs/archives/_index.md:22]:
```suggestion
Configure your Datadog account to forward all the logs ingested - whether [indexed][1] or not - to a cloud storage system of your own. Keep your logs in a storage-optimized archive for longer periods of time and meet compliance requirements while also keeping auditability for ad hoc investigations, with [Rehydration][2].
```

**Review by sarina-dd — APPROVED** [review]

File: content/en/logs/archives/_index.md [hunk 13]
```diff
@@ -249,25 +359,23 @@ Within the zipped JSON file, each event’s content is formatted as follows:
 
 **Note**: Archives only include log content, which consists of the message, custom attributes, and reserved attributes of your log events. The log tags (metadata that connects your log data to related metrics and traces) are not included.
```
Comment (estib) [inline content/en/logs/archives/_index.md:360]:
```suggestion
**Note**: Archives only include log content, which consists of the message, custom attributes, and reserved attributes of your log events. By default, the log tags (metadata that connects your log data to related metrics and traces) are not included.
```

File: content/en/logs/archives/_index.md [hunk 14]
```diff
@@ -249,25 +359,23 @@ Within the zipped JSON file, each event’s content is formatted as follows:
...
-[1]: /logs/archives/rehydrating/
+[1]: /logs/indexes/#exclusion-filters
+[2]: /logs/archives/rehydrating/
+[3]: /account_management/rbac/permissions/?tab=ui#logs_write_archives
+[4]: https://app.datadoghq.com/logs/pipelines/archives
+[5]: /account_management/rbac/permissions#logs_write_archives
+[6]: /account_management/rbac/permissions#logs_read_archives
+[7]: /account_management/rbac/permissions#logs_write_historical_view
+[8]: /account_management/rbac/permissions#logs_read_index_data
+[9]: /account_management/rbac/permissions#logs_read_data
```
Comment (estib) [inline content/en/logs/archives/_index.md:381]:
```suggestion
[9]: /account_management/rbac/permissions#logs_read_data
[10]: /logs/explorer/live_tail/
```

File: content/en/logs/archives/_index.md [hunk 15]
```diff
@@ -17,97 +17,241 @@ further_reading:
...
 
 {{< tabs >}}
 {{% tab "AWS S3" %}}
 
+If not already configured, set up the [AWS integration][1] for the AWS account that holds your S3 bucket.
+
+* In the general case, this involves creating a role that Datadog can use to integrate with AWS S3.
+* Specifically for AWS GovCloud or China accounts, use access keys as an alternative to role delegation.
+
+[1]: integrations/amazon_web_services/?tab=automaticcloudformation#setup
```
Comment (estib) [inline content/en/logs/archives/_index.md:49]:
```suggestion
[1]: /integrations/amazon_web_services/?tab=automaticcloudformation#setup
```

File: content/en/logs/archives/_index.md [hunk 16]
```diff
@@ -17,97 +17,241 @@ further_reading:
...
 
-You can [set a lifecycle configuration on your S3 bucket][3] to automatically transition your log archives to optimal storage classes. 
+{{< tabs >}}
+{{% tab "AWS S3" %}}
+
+You can [set a lifecycle configuration on your S3 bucket][1] to automatically transition your log archives to optimal storage classes.
+
+[Rehydration][1] supports all storage classes except for Glacier and Glacier Deep Archive. If you wish to rehydrate from archives in the Glacier or Glacier Deep Archive storage classes, you must first move them to a different storage class.
+
+[1]: /logs/archives/rehydrating/
```
Comment (estib) [inline content/en/logs/archives/_index.md:231]:
```suggestion
[1]: https://docs.aws.amazon.com/AmazonS3/latest/dev/how-to-set-lifecycle-configuration-intro.html
[2]: /logs/archives/rehydrating/
```

File: content/en/logs/archives/_index.md [hunk 17]
```diff
@@ -17,97 +17,241 @@ further_reading:
...
+
 #### Storage Class
 
-You can [set a lifecycle configuration on your S3 bucket][3] to automatically transition your log archives to optimal storage classes. 
+{{< tabs >}}
+{{% tab "AWS S3" %}}
+
+You can [set a lifecycle configuration on your S3 bucket][1] to automatically transition your log archives to optimal storage classes.
+
+[Rehydration][1] supports all storage classes except for Glacier and Glacier Deep Archive. If you wish to rehydrate from archives in the Glacier or Glacier Deep Archive storage classes, you must first move them to a different storage class.
```
Comment (estib) [inline content/en/logs/archives/_index.md:229]:
```suggestion
[Rehydration][2] supports all storage classes except for Glacier and Glacier Deep Archive. If you wish to rehydrate from archives in the Glacier or Glacier Deep Archive storage classes, you must first move them to a different storage class.
```

**Review by estib — APPROVED** [review]
This is beautiful!

Made a bunch of small suggestions -- mainly just correcting link references, a couple typos, and a few content suggestions that you can keep / remove as you like.

We should add the storage class and encryption content for GCP + Azure, but I'd suggest we ship this and add that in another PR.

**Pull request merged**